feature/bgp/policybase/otg_tests/import_export_multi/README.md

```diff
@@ -0,0 +1,207 @@
+# RT-7.11: BGP Policy - Import/Export Policy Action Using Multiple Criteria
+
+## Summary
+
+The purpose of this test is to verify a combination of bgp conditions using matching and policy nesting as well as and actions in a single BGP import policy.  Additional combinations may be added in the future as subtests.
+
+## Testbed type
+
+* [2 port ATE to DUT](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Testbed common configuration
+
+* Testbed configuration - Setup eBGP sessions and prefixes
+  * Generate config for 2 DUT and ATE ports where
+    * DUT port 1 connects to ATE port 1.
+    * DUT port 2 connects to ATE port 2.
+  * Configure ATE port 1 with an external type BGP session to DUT port 1
+    * DUT ASN 65000
+    * ATE port 1 ASN 65100
+    * ATE port 2 ASN 65200
+    * Advertise ipv4 and ipv6 prefixes from ATE port 1 to DUT port 1 using
+      the following communities:
+    * prefix-set-1 with 2 ipv4 and 2 ipv6 routes with communities [ "10:1" ]
+    * prefix-set-2 with 2 ipv4 and 2 ipv6 routes with communities [ "20:1" ]
+    * prefix-set-3 with 2 ipv4 and 2 ipv6 routes with communities [ "30:1" ]
+    * prefix-set-4 with 2 ipv4 and 2 ipv6 routes with communities [ "20:2", "30:3" ]
+    * prefix-set-5 with 2 ipv4 and 2 ipv6 routes with communities [ "40:1" ]
+    * prefix-set-6 with 2 ipv4 and 2 ipv6 routes with communities [ "50:1" ]
+    * Configure accept_route policy and apply as an export policy on the DUT
+      eBGP session to ATE port 2.
+
+* Configure the following community sets on the DUT:
+  * /routing-policy/defined-sets/bgp-defined-sets/ext-community-sets/ext-community-set/config
+    * name = "reject_communities"
+      * community-member = [ "10:1" ]
+    * name = "accept_communities"
+      * community-member = [ "20:1" ]
+    * name = "regex_community"
+      * community-member = [ "^30:.*$" ]
+    * name = "add_communities"
+      * community-member = [ "40:1", "40:2" ]
+    * name "my_community"
+      * community-member = [ "50:1" ]
+    * name = "add_comm_60"
+      * community-member = [ "60:1" ]
+    * name = "add_comm_70"
+      * community-member = [ "70:1" ]
+
+* Create an as-path-set on the DUT as follows
+  * /routing-policy/defined-sets/bgp-defined-sets/as-path-sets/as-path-set/config/
+    * as-path-set-name = "my_aspath"
+    * as-path-set-member = "65100"
+
+* Validate bgp sessions and traffic
+  * For IPv4 and IPv6 prefixes:
+    * Observe received prefixes at ATE port-2.
+  * Generate traffic from ATE port-2 to ATE port-1.
+  * Validate that traffic can be received on ATE port-1 for all installed
+    routes.
+
+## Procedure
+
+### RT-7.11.2 - Create a bgp policy containing the following conditions and actions
```

Comment:
```suggestion
### RT-7.11.1 - Create a bgp policy containing the following conditions and actions
```

Reply:
see above explanation for keeping the test environment / preconditions setup outside of the subtest(s)

Reply:
Ah I only had 7.11.2 on accident.  I will fix it.  I like the idea of a subtest for the config and a subtest for the verification, I'll do that.   Resolve this comment if you like the result.